### DISCUSSION TITLE
A script to remove a badge from READMEs

### DISCUSSION THREAD

**Discussion by RichardLitt** [octoherd/octoherd#15] [Ideas]
If I want to talk about how to do a new script, should I do that here? 

I'd like to remove a badge from all repos in an org (see https://github.com/orbitdb/orbit-db/issues/867). I think that octoherd might help.

**Comment by gr2m** [comment]
The discussion form in this repository is the best place to get involved 👍🏼 

> I'd like to remove a badge from all repos in an org (see [orbitdb/orbit-db#867](https://github.com/orbitdb/orbit-db/issues/867)). I think that octoherd might help.

Great candidate for an @octoherd script. 

For bootstrapping the project, use https://github.com/octoherd/create-octoherd-script (and let us know how it went). 

For removing a badge from a `README.md`, I think that @travi had a script that added/removed README badges? I did a quick search and found https://github.com/remarkjs/remark-strip-badges

**Reply by travi** [reply]
i've created a couple of remark plugins for removing specific badges from readmes. there is probably opportunity to make them a bit more efficient, but they should serve as reasonable examples, at least:

* https://github.com/form8ion/remark-remove-greenkeeper-badge
* https://github.com/form8ion/remark-remove-travis-ci-badge

i haven't combined them with octoherd quite yet, so i'm interested to see a combination like that once you get it working

**Comment by RichardLitt** [comment]
Hey! I haven't been able to allocate time for this. D: I hope someone else is able to, as I think it's a cool idea. I'm admitting to myself and ya'll that it's on my back backburner for now. Sorry to drop an idea and not get to it!
